### DISCUSSION TITLE
Is there a packet manager existing/planned?

### DISCUSSION THREAD

**Discussion by AlexNomadrg** [fnuecke/oc2#228] [Q&A] [answered]
Is there a packet manager existing/planned? And can you install X11 on that thing?

**Comment by atirutw** [comment, marked as answer]
The default distro does not have any package manager and I don't think it's planned. Best case scenario it gets implemented after they got external internet connection to work.

**Reply by spike11302000** [reply]
it theoretically possible to run X11/wayland as there is a framebuffer using the projector its just havent been tried yet so not sure if it will work. theres also no mouse support as well so you would need to use a window manager that works with only the keyboard, like i3 or sway.

**Reply by polyzium** [reply]
TCC is included inside Sedna.

**Reply by spike11302000** [reply]
> we need c compiler first

you can cross compile on a linux pc then use the file export/import card to transfer the files
